oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -1035,7 +1026,17 @@ impl Runtime {
             OakStatus::ErrInvalidArgs
         })?;
 
-        debug!("{:?}: start node instance {:?}", node_id, new_node_id);
+        let node_privilege = instance.get_privilege();
+
+        self.node_configure_instance(new_node_id, &new_node_name, label, &node_privilege);
```

Comment:
pre-existing: Is it possible to give this function a better name, like `register_node_info`? I find it a bit confusing.

Reply:
I think there is a function `add_node_info` already, and TBH I agree with you this does not really add much value. I'll try and simplify both in a refactoring.